### PR TITLE
Move prepare objects to a background job

### PR DESCRIPTION
**Pull request by jcoyne**
~~DO NOT MERGE without approval from Andrew~~   approved!

Ref #1465

### Before:
Screen Shot 2019-08-20 at 2 31 40 PM

Screen Shot 2019-08-20 at 2 30 36 PM

### After:

Screen Shot 2019-08-20 at 1 37 29 PM

Screen Shot 2019-08-20 at 1 38 19 PM

### CODE REVIEW

File: app/jobs/manage_catkey_job.rb [hunk 1]
```diff
@@ -10,7 +10,8 @@ class ManageCatkeyJob < GenericJob
   # A job that allows a user to specify a list of pids and a list of catkeys to be associated with these pids
   # @param [Integer] bulk_action_id GlobalID for a BulkAction object
   # @param [Hash] params additional parameters that an Argo job may need
-  # @option params [Array] :pids required list of pids
+  # @option params [Array] :pids required list of
+
```
Comment (ndushay) [inline app/jobs/manage_catkey_job.rb]: errant return added?

File: spec/features/bulk_screen_spec.rb [hunk 2]
```diff
@@ -17,8 +17,8 @@
 
     find(:xpath, "//span[@class='bulk_button'][text()='Get druids from search'][not(@disabled)]").click
 
-    expect(find_button('Prepare objects')).to be_truthy
-    expect(page).to have_button('Prepare objects', disabled: false)
+    # expect(find_button('Prepare objects')).to be_truthy
+    # expect(page).to have_button('Prepare objects', disabled: false)
```
Comment (ndushay) [inline spec/features/bulk_screen_spec.rb]: none of the rest of this PR comments out the old way -- any reason you're doing that here?

Comment (jcoyne) [reply spec/features/bulk_screen_spec.rb]: I've removed these now.

Comment (ndushay) [reply spec/features/bulk_screen_spec.rb]: github PR seems to think they are still there ...

Comment (jcoyne) [reply spec/features/bulk_screen_spec.rb]: Okay, I *really really* did it this time.

**Comment by ndushay** [comment]
PO approved!

**Comment by jmartin-sul** [comment]
🎉

**Pull request merged**